source/extended-json/extended-json-query.md

```diff
@@ -0,0 +1,56 @@
+# Extended JSON Query
+
+## Abstract
+
+Extended JSON query variation is a superset of Extended JSON. It is
+intended to be used for serializing queries that can be specified in
+the mongo shell. It supplements the Extended JSON syntax with support
+for the following features:
+
+- Trailing commas after the last value in an object
+
+- NumberLong values in quotation marks (aka "double quote")
+  - Apostrophe (aka "single quote") is not supported
+
+- Dates constructed with `new Date()` including an `ISODate` format
+  - Including support for pre-epoch dates
+
+- Regex pattern enclosed in slashes
+
+## Examples
```

Comment:
I expected this to be much more specific about the format. Like, can we describe the exact representation for each BSON type? How is DBPointer represented? How is BinData and our various BinData subtypes represented? I guess this question is closely related to whether it's true that this format is indeed supposed to be an extension of extended JSON, which was not my understanding when we first discussed this.